source/scss/_library/_module.footer.scss

```diff
@@ -56,11 +56,10 @@
     margin-bottom: $space;
   }
 
-  .gothamist-letters {
-    fill: $c-white;
-  }
-  .gothamist-buildings {
-    fill: $c-dark-bg;
```

Comment:
why no longer giving these a fill?

Reply:
Since these are really Gothamist specific, I moved these to the Gothamist theme partial.